### PR TITLE
add load 3d node support

### PR DESCRIPTION
**Pull request by jtydhr88**
Hi,
I implment a node to load local 3d models, here is FE change:
https://github.com/Comfy-Org/ComfyUI_frontend/pull/1563

which needs this BE changes.

here are the introduction video as your reference:
1. basic load feature

https://github.com/user-attachments/assets/f9f1a7f4-5bd2-4a96-b9d7-681b2d7f68f3

3. showGrid, cameraType, View

https://github.com/user-attachments/assets/3c539495-6a35-49eb-84fc-14d5dd8c55c4

5. material, lightIntensity

https://github.com/user-attachments/assets/dbc4dd0b-d387-4ec0-b276-5e966ccc6794

7. fbx and animation

https://github.com/user-attachments/assets/a615f99a-7555-477c-beb4-d1d885bf093b

9. up-direction

https://github.com/user-attachments/assets/40e9f61a-91cf-4e7a-8b62-e3dd1a59e9d5

11. multi-models (video file is too big)
12. supported-formats (video file is too big)
13. preview-3d-node

https://github.com/user-attachments/assets/ec69325a-78ea-43b6-beaf-acc568f37505

### CODE REVIEW

**Review by huchenlei — DISMISSED** [review]

**Comment by huchenlei** [comment]
@comfyanonymous  PTAL, the frontend PRs are already merged. We should have this PR merged on next frontend sync this Friday.

**Comment by yovsac** [comment]
This node can be a revolution on how we use comfy. Is it going to be implemented any time soon??

**Comment by huchenlei** [comment]
@comfyanonymous PTAL. I do think @jtydhr88 should become the owner of these nodes, i.e. we don't try to understand every detail of the implementation and when issue pop-up we just do necessary reverts and assign issue to the owner.

File: comfy_extras/nodes_load_3d.py [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+        imagepath = folder_paths.get_annotated_filepath(image)
+
+        load_image_node = nodes.LoadImage()
+
+        output_image, output_mask = load_image_node.load_image(image=imagepath)
+
+        return output_image, output_mask, model_file,
+
+class Preview3D():
+    @classmethod
```
Comment (comfyanonymous) [inline comfy_extras/nodes_load_3d.py]: Does this node need to be in the backend? Can it be a frontend only node?

Comment (jtydhr88) [reply comfy_extras/nodes_load_3d.py]: good catch, I think so.
let me check and give you back soon

Comment (huchenlei) [reply comfy_extras/nodes_load_3d.py]: > good catch, I think so. let me check and give you back soon

Let's just remove that Preview3D node for now and get this PR merged. We can decide how to handle that Preview3D node later.

Comment (jtydhr88) [reply comfy_extras/nodes_load_3d.py]: removed, I will work on how to implement Preview3D on FE only

**Review by huchenlei — APPROVED** [review]

**Comment by yovsac** [comment]
Great news! Does this mean we can start using it when we update comfy ?

**Comment by AndyWu2015** [comment]
so cool， just want to try comfyui. thanks Terry. （太酷啦，正好想试着用用comfyui。感谢Terry）

**Comment by jtydhr88** [comment]
> Great news! Does this mean we can start using it when we update comfy ?

Yes! please update to the latest!

**Pull request merged**